packages/core/src/middleware/koa-log.test.ts

```diff
@@ -44,9 +44,11 @@ describe('koaLog middleware', () => {
       log,
     };
     ctx.request.ip = ip;
+    const additionalMockPayload: LogPayload = { foo: 'bar' };
 
     const next = async () => {
       ctx.log(type, mockPayload);
+      ctx.log(type, additionalMockPayload);
```

Comment:
What is this line used for?

Reply:
need to call `ctx.log(...)` twice to cover the line below
<img width="789" alt="image" src="https://user-images.githubusercontent.com/10594507/181181856-3b5910bf-c046-4cba-8ff0-654d21cc1428.png">